Update Cover and Intro Page (#14087)
dotnet/docs: docs/architecture/cloud-native/index.md

# Architecting Cloud Native .NET Applications for Azure

> ![cover image](./media/cover-preview.png)

Reviewer: instead of creating a new cover, shouldn't you just replace the temp file you have in the directory now?

Reviewer 2: I agree. It's easier to replace the image when the books go out of Preview.

>
> **Rob Vettor** - Microsoft - Principal Cloud System Architect/IP Architect - [RobVettor.com](http://robvettor.com)

Participants and Reviewers:

> > **Cesar de la Torre**, Sr. PM, .NET product team, Microsoft Corp.

Reviewer: Can we be consistent if we're going to refer to Microsoft with or without Corp?

Reviewer 2: ```suggestion
> **Cesar de la Torre**, Principal Program Manager, .NET team, Microsoft
```

>
> **Nish Anil**, Sr. Program Manager, .NET team, Microsoft

Editors:

> > **Maira Wenzel**, Content Developer, .NET team, Microsoft

Reviewer: if you're adding seniority, you might as well add mine 😉 thanks for including me!

```suggestion
> **Maira Wenzel**, Sr. Content Developer, .NET team, Microsoft
```

## Who should use this guide

The audience for this guide is mainly developers, development leads, and architects who are interested in learning how to build applications designed for the cloud.

A secondary audience is technical decision-makers who plan to choose whether to build their applications using a cloud-native approach.

## How you can use this guide

> This guide begins by defining cloud-native and introducing a reference application built using cloud-native principles and technologies. Beyond these first two chapters, the rest of the book is broken up into specific chapters focused on topics common to most cloud-native applications. You can jump to any of these chapters to learn about cloud-native approaches to:

Reviewer: ```suggestion
This guide begins by defining cloud native and introducing a reference application built using cloud-native principles and technologies. Beyond these first two chapters, the rest of the book is broken up into specific chapters focused on topics common to most cloud-native applications. You can jump to any of these chapters to learn about cloud-native approaches to:
```

- Data and data access
- Communication patterns
- Scaling and scalability
- Application resiliency
- Monitoring and health
- Identity and security
- DevOps

> This guide is available both in PDF form and online. Feel free to forward this document or links to its online version to your team to help ensure common understanding of these topics. Most of these topics benefit from a consistent understanding of the underlying principles and patterns, as well as the tradeoffs involved in decisions related to these topics. Our goal with this document is to equip teams and their leaders with the information they need to make well-informed decisions for their applications' architecture, development, and hosting.

Reviewer: missing hyphen in trade-offs

```suggestion
This guide is available both in PDF form and online. Feel free to forward this document or links to its online version to your team to help ensure common understanding of these topics. Most of these topics benefit from a consistent understanding of the underlying principles and patterns, as well as the trade-offs involved in decisions related to these topics. Our goal with this document is to equip teams and their leaders with the information they need to make well-informed decisions for their applications' architecture, development, and hosting.
```

> ## References

Reviewer 2: We should drop this.
